src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -0,0 +1,355 @@
+              renderInput={(params: any) => (
+                <TextField
+                  {...params}
+                  error={!!errors.region}
+                  helperText={errors.region ? errors.region.message : ''}
+                  placeholder="cloud region"
+                />
+              )}
+              {...field}
+              onChange={(_event: any, change: any) => {
+                field.onChange(change);
+              }}
+            />
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={12} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-stewards"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.stewards })}
+        >
+          Stewards
+        </label>
+        <div className={classes.formFieldDescription}>
+          A Steward, or Data Owner, is the person who created the dataset. While they are ultimately
+          liable for the data, they can assign the hands-on data management to another person by
+          assigning the Custodian role.
+        </div>
+        <Controller
+          name="stewards"
+          control={control}
+          rules={{
+            validate: {
+              isEmail: (values: string[]) => {
+                const emailErrors = _.filter(values, (v: string) => !isEmail(v));
+                return emailErrors.length === 0 || `Invalid emails: "${emailErrors.join('", "')}"`;
+              },
+            },
+          }}
+          render={({ field }) => (
+            <Autocomplete
+              id="dataset-stewards"
+              freeSolo
+              multiple
+              options={[]}
+              className={classes.formInput}
+              renderInput={(params: any) => (
+                <TextField
+                  {...params}
+                  error={!!errors.stewards}
+                  helperText={errors.stewards ? errors.stewards.message : ''}
+                  placeholder="username@email.com"
+                />
+              )}
+              {...field}
+              onChange={(_event: any, change: any) => {
+                field.onChange(change);
+              }}
+            />
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={12} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-custodians"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.custodians })}
+        >
+          Custodians*
+        </label>
+        <div className={classes.formFieldDescription}>
+          The Custodian role is defined on a dataset. Someone may be the Custodian for one or more
+          studies. A Custodian is responsible for creating data snapshots over datasets and
+          controlling access to those snapshots.
+        </div>
+        <Controller
+          name="custodians"
+          control={control}
+          rules={{
+            validate: {
+              isEmail: (values: string[]) => {
```

Comment:
(See above comment, this is actually an example where that reusability would be very powerful.)